scripts/mkcloud

```diff
@@ -15,6 +15,13 @@
 # 2. This tool relies on the script qa_crowbarsetup.sh
 # 3. Please 'export' environment variables according to your needs.
 
+: ${virtualcloud:=virtual}
+
+: ${log_dir:=/var/log/mkcloud/$virtualcloud}
```

Comment:
:-1: 
Oh there can be many clouds with the (internal) name 'virtual'. 
The differentiating name is held in the `$cloud` variable.